<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB Connection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="l_manager">
        <property name="text">
         <string>Choose Manager:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_database_2">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_database_5">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="l_database_3">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="l_database">
        <property name="text">
         <string> Database:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="cb_manager"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_host"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="le_username"/>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="le_database"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="le_pass">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_database_4">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_port"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>horizontalSpacer_2</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>horizontalSpacer_3</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>horizontalSpacer_4</zorder>
   <zorder>horizontalSpacer_5</zorder>
   <zorder>horizontalSpacer_6</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>layoutWidget_3</zorder>
   <zorder>layoutWidget_4</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>btn_connect</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cb_manager</tabstop>
  <tabstop>le_host</tabstop>
  <tabstop>le_port</tabstop>
  <tabstop>le_username</tabstop>
  <tabstop>le_pass</tabstop>
  <tabstop>le_database</tabstop>
  <tabstop>btn_connect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
